fury/tests/test_primitive.py

```diff
@@ -77,6 +77,16 @@ def test_vertices_primitives_triangularprism():
     npt.assert_equal(vertices.max(), 1/2)
 
 
+def test_vertices_primitives_rhombicuboctahedron():
+    # Testing the default vertices of the primitive rhombicuboctahedron.
+    vertices, _ = fp.prim_rhombicuboctahedron()
+    shape = (24, 3)
+    npt.assert_equal(vertices.shape, shape)
+    npt.assert_equal(np.mean(vertices, axis=0), 0)
+    npt.assert_equal(vertices.min(), -0.5)
+    npt.assert_equal(vertices.max(), 0.5)
+
```

Comment:
You can remove this. this is a duplicate. This is already done line 13

Reply:
Thanks for pointing this out, removed that. @skoudoro